core/src/main/java/com/sequenceiq/cloudbreak/service/rdsconfig/AbstractRdsConfigProvider.java

```diff
@@ -72,9 +72,9 @@
                 }
                 String dbName = getDb();
                 postgres.put("database", dbName);
-                postgres.put("user", getDbUser());
+                postgres.put("user", rdsConfig.getConnectionUserName());
```

Comment:
Are we sure we want to use the same username for all databases?
Previously this was based on RdsProvider type.

Reply:
This doesn't use the same username for all databases. Above this line, the method fetches the RDSConfig object for the provider type (based on `getRdsType()`). That object, in turn, is saved earlier in the datalake creation, by the `createPostgresRdsConfigIfNeeded` method which is below this line in the file. The user for the RDS provider implementation is used there, so each RDSConfig object already receives a username specific to the provider type.

```
newRdsConfig = dbServerConfigurer.createNewRdsConfig(stack, cluster, getDb(), getDbUser(), getRdsType());
```

This change is here now so that the value of "user" written to the salt pillar is in *username@short-hostname* format, which is necessary for scm_prepare_database.sh to work with it.

Reply:
To avoid further rebase problems, I'm going to close out this question / discussion, but do hit me up if you need more information. I will have one more small PR after this as well.